test/meta.js

```diff
@@ -123,7 +123,7 @@ test('emit unsubscribe event', function (t) {
   t.plan(6)
 
   var broker = aedes()
-  var s = connect(setup(broker), { clientId: 'abcde' })
+  var s = connect(setup(broker), { clean: true, clientId: 'abcde' })
```

Comment:
can you add another test that fails if the event is emitted when clean is false and the connection ends abruptly?

Reply:
foget about it, it's already added.